src/frontend/utils/chat/chat.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { converse } from 'utils/window';
```

Comment:
In the commit message you have to use the use the imperative, present tense.

`Add utilitaries related to anonymous nicknames and tokens generation.`

Reply:
new commit with correct message is now [d15246dda06464b53f29942cc7506b2b0dbdec73](https://github.com/openfun/marsha/pull/1276/commits/d15246dda06464b53f29942cc7506b2b0dbdec73)